packages/shared/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@acs/shared",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc"
+  },
+  "dependencies": {
+    "mongoose": "^5.10.6"
```

Comment:
I wonder if it would be possible to move `mongoose` to dev deps. I think we're only using it for types

Reply:
sgtm. Should be fine. updated in 26a090b84e7934dd9f981a867fe3ff89d09bf9d3